devices/esp32c3.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "category": "ESP32",
+    "name": "Timer C3",
+    "targets": [
+      {
+        "flashingMethod": "UART",
+        "name": "esp32c3"
```

Comment:
Not adding support for s3 like the firmware?

Reply:
added